app/models/appliance_comment.rb

```diff
@@ -15,6 +16,10 @@ def compact_body
     body.truncate(30)
   end
 
+  def thumb_nail
```

Comment:
It is practical to define variants in the model, because this way we can have uniform sizes in all views.